docs/tasks/scaleworkload.rst

```diff
@@ -0,0 +1,50 @@
+.. _scaleworkloadexample:
+
+Using ScaleWorkload function with output artifact
+-------------------------------------------------
+
+Traditionally, when ``ScaleWorkload`` functions were used to scale down the
+workload, it was required to get the original replica count of the workload and
+set it as output artifact. So that it can later be used to scale up the
+workload to same number of replicas.
+
+After the new changes, the ``ScaleWorkload`` function automatically sets the
```

Comment:
IMO it doesn't make sense or add any value mentioning history about how it used to behave earlier.

Reply:
ok, makes sense.
I wanted to mention that just so that, if people come across a blueprint or have a blueprint that is setting replica count in legacy way, they should know why that's happening.
But I can remove it from here, if you think that's better.

Reply:
I don't think it would matter how it was used to work earlier once we override the functionality.